src/screens/shared/farmProfileEdit/FarmProfileEdit.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+import BackButton from '@components/BackButton';
+
+import FarmProfileEditForm from './FarmProfileEditForm';
+import FarmProfileEditDropdown from './FarmProfileEditDropdown';
+import FarmProfileEditGapStatus from './FarmProfileEditGapStatus';
+
+const styles = {
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    margin: 64,
+    padding: 72,
+    backgroundColor: 'white',
+    maxWidth: 1680,
+    width: '100%'
+  }
+};
+
+class FarmProfileEdit extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      formValues: {
+        physicalState: 0,
+        mailState: 0
+      },
+      dropdownValues: {
+        gapContact: 0,
+        foodHubAffiliation: 0
+      },
+      gapCertificationValues: {
+        referred: 0,
```

Comment:
Currently, we do not add all our state variables in our parent state (it is hard to read and manage). We let our child components define their local state and use `onChange` to update the parent state. 

Ideally, when Airtable data is pulled, it also initializes all our state variables implicitly. The only initialized values are for dropdowns that need to be not `undefined` when it first renders.

Reply:
The above comment may be useful for documentation.